source/extensions/filters/http/dynamic_modules/filter.cc

```diff
@@ -7,13 +7,26 @@ namespace Extensions {
 namespace DynamicModules {
 namespace HttpFilters {
 
+DynamicModuleHttpFilter::~DynamicModuleHttpFilter() { destroy(); }
+
 void DynamicModuleHttpFilter::initializeInModuleFilter() {
   in_module_filter_ = config_->on_http_filter_new_(config_->in_module_config_, thisAsVoidPtr());
 }
 
 void DynamicModuleHttpFilter::onStreamComplete() {}
 
-void DynamicModuleHttpFilter::onDestroy() { config_->on_http_filter_destroy_(in_module_filter_); };
+void DynamicModuleHttpFilter::onDestroy() {
+  ENVOY_LOG_MISC(trace, "onDestroy called");
```

Comment:
not sure if I want to keep this log but at least can you use 

`ENVOY_LOG_TO_LOGGER(Envoy::Logger::Registry::getLog(Envoy::Logger::Id::dynamic_modules), trace`...

Reply:
removed